code/game/objects/items/weapons/defib.dm

```diff
@@ -452,16 +452,18 @@
 						update_icon(UPDATE_ICON_STATE)
 						defib.cooldowncheck(user)
 						return
-				if(H.stat == DEAD)
+				if(H.stat == DEAD || HAS_TRAIT(H, TRAIT_FAKEDEATH))
 					var/health = H.health
+					var/brain = H.get_int_organ(/obj/item/organ/internal/brain)
 					M.visible_message("<span class='warning'>[M]'s body convulses a bit.")
 					playsound(get_turf(src), "bodyfall", 50, 1)
 					playsound(get_turf(src), 'sound/machines/defib_zap.ogg', 50, 1, -1)
 					for(var/obj/item/organ/external/O in H.bodyparts)
 						total_brute	+= O.brute_dam
 						total_burn	+= O.burn_dam
 					ghost = H.get_ghost(TRUE) // We have to double check whether the dead guy has entered their body during the above
-					if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !HAS_TRAIT(H, TRAIT_HUSK) && !HAS_TRAIT(H, TRAIT_BADDNA) && H.blood_volume > BLOOD_VOLUME_SURVIVE && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)) && !(signal_result & COMPONENT_BLOCK_DEFIB))
+					// god this sucks. if you update something here, make sure to update it (read: duplicate it) in the borg stuff below as well.
+					if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !HAS_TRAIT(H, TRAIT_HUSK) && !HAS_TRAIT(H, TRAIT_BADDNA) && H.blood_volume > BLOOD_VOLUME_SURVIVE && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)) && !(signal_result & COMPONENT_BLOCK_DEFIB) && !HAS_TRAIT(M, TRAIT_FAKEDEATH) && brain)
```

Comment:
Depending on which gets merged first (https://github.com/ParadiseSS13/Paradise/pull/19987 or this), this is gonna conflict

Reply:
👍 no worries